src/test/kotlin/uk/gov/dluhc/registercheckerapi/config/MySQLContainerConfiguration.kt

```diff
@@ -4,7 +4,7 @@ import org.testcontainers.containers.MySQLContainer
 
 class MySQLContainerConfiguration : MySQLContainer<MySQLContainerConfiguration>(MYSQL_IMAGE) {
     companion object {
-        private const val MYSQL_IMAGE = "mysql:8"
+        private const val MYSQL_IMAGE = "mysql:8.2"
```

Comment:
We bump this to fix an error with running tests in the pipeline. See https://softwire.slack.com/archives/C05KAM40NUU/p1705685631577619